app/assets/src/components/ui/containers/Modal.jsx

```diff
@@ -31,7 +34,8 @@ Modal.propTypes = {
   ]).isRequired,
   onClose: PropTypes.func,
   title: PropTypes.string,
-  trigger: PropTypes.node.isRequired
+  trigger: PropTypes.node,
```

Comment:
If trigger makes no sense... should we just pass a dummy value and remove the prop from our component?

Reply:
That sounds good.